test/parser.test.js

```diff
@@ -747,7 +747,7 @@ describe('parser', () => {
         first: '',
         second: '',
         third: {
-          first: 'Hello <1>{{name}}</1>, you have {{count}} unread message. <5>Go to messages</5>.',
+          first: 'Hello <strong>{{name}}</strong>, you have {{count}} unread message. <5>Go to messages</5>.',
```

Comment:
I believe this is a mistake, it shouldn't parse a basic tag here, will try to fix this.

Reply:
Fixed